language/break_spec.rb

```diff
@@ -60,6 +60,13 @@
       ScratchPad.recorded.should == [:a, :za, :xa, :zd, :aa, :zb]
     end
   end
+
+  describe "from another thread" do
+    it "raises a LocalJumpError when getting the value from another thread" do
+      lambda { @program.break_in_another_thread }.should raise_error(LocalJumpError)
+      ScratchPad.recorded.should == [:a, :b]
```

Comment:
This looks great but it's not clear whether Thread#value raises the error or just break itself when executed.
Could you test that too?